readme.md

```diff
@@ -201,15 +211,15 @@ Once installation is working, take a look at the existing code to make sure you
 * **Question:** What happens? Why?
 
 * **Your Answer:**
-
+the bearer token is split and verified through comparing with the secret key.  Then the user id is pulled from the payload
 ---
 
 - [ ] In order to successfully access this route, we will need to send over the token in the HTTP Authorization Header. The typical way to do this is by sending a [Bearer token](https://security.stackexchange.com/questions/108662/why-is-bearer-required-before-the-token-in-authorization-header-in-a-http-re). To do this in Postman, go to the "Authorization" tab, select "Bearer Token" as the Type, and then enter your token.
 
 * **Question:** What happens? Why?
 
 * **Your Answer:**
-
+the bearer token is encrypted in an https request and returned to the api
```

Comment:
- Status 200 received with response which identifies the user associated to the token. This time we are authenticated/authorized by way of the token submitted along with the HTTP request.